datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaTransportProvider.java

```diff
@@ -56,6 +56,7 @@
   private final Meter _eventWriteRate;
   private final Meter _eventByteWriteRate;
   private final Meter _eventTransportErrorRate;
+  private final boolean _preserveSourceEventTimestamp;
```

Comment:
For consistency with the metadata name "preserveEventSourceTimestamp", suggest renaming this variable.

_preserveSourceEventTimestamp rename to _preserveEventSourceTimestamp

Reply:
done